### PR TITLE
Configure the logs folder for WsAgent and ExecAgent

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?

Configure the logs folder for WsAgent and ExecAgent to use the volume added with https://github.com/eclipse/che/pull/5708

### What issues does this PR fix or reference?
https://github.com/redhat-developer/rh-che/issues/145

#### Changelog
Configure the logs folder for WsAgent and ExecAgent to use persistent volume /workpace-logs

Change-Id: I24e9e4feefd3d80b66f2719252036432a2274edc
Signed-off-by: Florent BENOIT <fbenoit@redhat.com>

### CODE REVIEW

File: builds/fabric8-che/assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java [hunk 1]
```diff
@@ -174,7 +174,7 @@ protected void configure() {
         launchers.addBinding().to(SshAgentLauncher.class);
 
         bindConstant().annotatedWith(Names.named("machine.ws_agent.run_command"))
-                      .to("export JPDA_ADDRESS=\"4403\" && ~/che/ws-agent/bin/catalina.sh jpda run");
+                      .to("export JPDA_ADDRESS=\"4403\" && export CHE_LOGS_DIR=\"/workspace-logs/${CHE_MACHINE_NAME}-ws-agent\" && env && ~/che/ws-agent/bin/catalina.sh jpda run");
```
Comment (sunix) [inline builds/fabric8-che/assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java]: is `env` command necessary ? or just for debugging ?

Comment (benoitf) [inline builds/fabric8-che/assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java]: ah yes it's a mistake or rebasing :-)

Comment (benoitf) [inline builds/fabric8-che/assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java]: fixed

File: builds/fabric8-che/assembly/pom.xml [hunk 2]
```diff
@@ -21,7 +21,7 @@
         <module>fabric8-stacks</module>
         <module>assembly-wsagent-war</module>
         <module>assembly-wsagent-server</module>
-        <module>assembly-ide-war</module>
```
Comment (sunix) [inline builds/fabric8-che/assembly/pom.xml]: why ?

**Pull request merged**